Library/Formula/rocksdb.rb

```diff
@@ -20,6 +18,9 @@ class Rocksdb < Formula
   depends_on "snappy"
   depends_on "lz4"
 
+  # fix the weird library naming scheme in rocksdb Makefile
+  patch :DATA
```

Comment:
Sorry, I mean use `https://github.com/facebook/rocksdb/pull/670.patch` as a `patch do` block (unless it doesn't apply cleanly). Either way it'd be good to have that linked in a comment so there's more context. Thanks!